README.md

```diff
@@ -84,6 +84,26 @@ These examples will help you understand how to use this library. You can also br
 
 Once you initialize a `client`, you can call methods to make requests to the Okta API. Most methods are grouped by the API endpoint they belong to. For example, methods that call the [Users API](https://developer.okta.com/docs/api/resources/users) are organized under `client.User`.
 
+
+## Connection Retry / Rate Limiting
+By default this SDK will retry requests that are return with a 429 exception. To disable this functionality set `OKTA_CLIENT_REQUESTTIMEOUT` and `OKTA_CLIENT_RATELIMIT_MAXRETRIES` to 0.
+
+Setting only one of the values to zero will disable that check. Meaning, by default, four retry attempts will be made. If you set `OKTA_CLIENT_REQUESTTIMEOUT` to 45 seconds and `OKTA_CLIENT_RATELIMIT_MAXRETRIES` to 0. This SDK will continue to retry indefinitely for 45 seconds. If both values are non zero, this SDK will attempt to retry until either of the conditions are met (not both).
```

Comment:
Active voice:
Setting only one of the values to zero ~~will disable~~ **disables** that check.  Meaning, by default, four retry attempts ~~will be~~ **are** made. If you set `OKTA_CLIENT_REQUESTTIMEOUT` to 45 seconds and `OKTA_CLIENT_RATELIMIT_MAXRETRIES` to 0. This SDK ~~will continue~~ **continues** to retry indefinitely for 45 seconds. If both values are non zero, this SDK ~~will attempt~~ **attempts**  to retry until either of the conditions are met (not both).